pkg/controller/user-cluster-controller-manager/resources/reconciler.go

```diff
@@ -71,16 +71,24 @@ func (r *reconciler) reconcile(ctx context.Context) error {
 	if err != nil {
 		return fmt.Errorf("failed to get userSSHKeys: %v", err)
 	}
-	cloudConfig, err := r.cloudConfig(ctx)
+	cloudConfig, err := r.cloudConfig(ctx, resources.CloudConfigConfigMapName)
 	if err != nil {
 		return fmt.Errorf("failed to get cloudConfig: %v", err)
 	}
+	var CSICloudConfig []byte
```

Comment:
Tiny nit: style change

```suggestion
	var csiCloudConfig []byte
```